tool/tctl/common/resource_command_test.go

```diff
@@ -1371,17 +1372,29 @@ func TestCreateResources(t *testing.T) {
 	process := testenv.MakeTestServer(t, testenv.WithLogger(utils.NewSlogLoggerForTests()))
 	rootClient := testenv.MakeDefaultAuthClient(t, process)
 
+	// tctlGetAllValidations allows tests to register post-test validations to validate
+	// that their resource is present in "ttcl get all" output.
+	// This allows running test rows instead of the whole test table.
+	var tctlGetAllValidations []func(t *testing.T, out string)
+
```

Comment:
Sorry for the noisy changes in the table test but I got fed up by the test failing if I only ran a single row. Now the individual rows can be ran.